frontend/src/templates/Field/Radio/RadioField.tsx

```diff
@@ -87,9 +87,18 @@ export const RadioField = ({
               if (value === RADIO_OTHERS_INPUT_VALUE) trigger(othersInputName)
             }}
             aria-label={schema.title}
+            aria-invalid={
+              !!get(errors, radioInputName) || !!get(errors, othersInputName)
+            }
+            aria-required={schema.required}
           >
             {schema.fieldOptions.map((option, idx) => (
-              <Radio key={idx} value={option} {...(idx === 0 ? { ref } : {})}>
+              <Radio
+                key={idx}
+                value={option}
+                {...(idx === 0 ? { ref } : {})}
+                isRequired={false}
```

Comment:
Can we add a comment to explain why we set `isRequired` in places that we do this override so that we don't delete this by accident in future hehe 🙏

Reply:
Added!